modules/enabling-gpu-support-in-data-science.adoc

```diff
@@ -4,7 +4,7 @@
 = Enabling GPU support in {productname-short}
 
 [role='_abstract']
-Optionally, to ensure that your data scientists can use compute-heavy workloads in their models, you can enable graphics processing units (GPUs) in {productname-short}. To enable GPUs on OpenShift, you must install the NVIDIA GPU Operator. As a prerequisite to installing the NVIDIA GPU Operator, you must install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/specialized_hardware_and_driver_enablement/node-feature-discovery-operator[Node Feature Discovery Operator]. For information about how to install these operators, see link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on Red Hat OpenShift Container Platform^] in the NVIDIA documentation.
+Optionally, to ensure that your data scientists can use compute-heavy workloads in their models, you can enable graphics processing units (GPUs) in {productname-short}. To enable GPUs on OpenShift, you must install the NVIDIA GPU Operator. As a prerequisite to installing the NVIDIA GPU Operator, you must install the link:https://access.redhat.com/documentation/en-us/openshift_container_platform/{ocp-latest-version}/html/specialized_hardware_and_driver_enablement/node-feature-discovery-operator[Node Feature Discovery Operator]. For information about how to install these operators, see link:https://docs.nvidia.com/datacenter/cloud-native/openshift/latest/index.html[NVIDIA GPU Operator on {org-name} OpenShift Container Platform^] in the NVIDIA documentation.
```

Comment:
See: [NVIDIA GPU Operator on {org-name} OpenShift Container Platform^]

Is the ^ meant to be there?

Reply:
It looks like it could be intentional from the original change (https://github.com/opendatahub-io/opendatahub-documentation/pull/56/files) but could check that with the rest of the NVIDIA GPU details in a new ticket?